app/scripts/auth/controller.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+module.examples[
+  '$scope',
+  'loginService',
+  function loginControler($scope, loginService) {
+    $scope.user = {
+      email: '',
```

Comment:
In markup I used username as value. For consistency we should stick to only one of them.